app/core/Engine/Engine.ts

```diff
@@ -470,6 +470,15 @@ export class Engine {
       }),
       state: initialState.LoggingController,
     });
+
+    // TODO: Move this to `network-controller`
+    const enableOrDisableRpcFailover = (isRpcFailoverEnabled: Json) => {
```

Comment:
S: when the state is switched form one to the other, we usually use the term toggle:

```suggestion
    const toggleRpcFailover = (isRpcFailoverEnabled: Json) => {
```

Reply:
Makes sense. Updated in [f2e5e9e](https://github.com/MetaMask/metamask-mobile/pull/21168/commits/f2e5e9e3cc200f825fa8acbbc1fe79d5fe46a8c2).